src/frontend/README.md

```diff
@@ -44,14 +44,25 @@ else.
 
 ### Metrics
 
-- Automatic Instrumentation (Agent): :large_blue_diamond:
-- Automatic Instrumentation (OTel Operator): :red_circle:
-- Instrumentation Libraries  : :red_circle:
-- Manual Metric Creation:  :red_circle:
-- Collector Metric Transformation::red_circle:
-- Push Metrics: :red_circle:
-- SLO Metrics: :red_circle:
-- Multiple Metric Instruments: :red_circle:
+- [Automatic Instrumentation (Language
+  Agent)](https://github.com/open-telemetry/opentelemetry-java-instrumentation#about):
```

Comment:
It's a bit strange to have the frontend (which is implemented in Golang) `README.md` link to a Java related doc.

Reply:
Go doesn't have the concept of auto instrumentation using a go agent so I figured it made more sense to link to the concept in general. Feel free to propose a better link

Reply:
This is just passing the buck down the line, but I feel like this is a concept that should be defined within the core OpenTelemetry docs themselves.  Perhaps in this section: https://opentelemetry.io/docs/concepts/instrumenting/#automatic-instrumentation

Reply:
Yeah it'd be nice to point to generic definitions of items. It's easier for things like instrumentation libraries

Reply:
And I would suggest that we remove "(Language Agent)" because 1) it's not defined 2) it is super confusing.

Reply:
I personally find just Agent more confusing. Is the agent the collector, is it a language agent like Java etc.

Reply:
My understanding is that there is no such concept called "language agent" in OpenTelemetry.

The language agnostic concept is called "automatic instrumentation" and is defined here https://opentelemetry.io/docs/concepts/instrumenting/#automatic-instrumentation.
For Java, a special technology is used based on JVM agent, and some vendors call it "Java Agent" due to historical or whatever reasons.

Reply:
Updated the links to be more generic. I see the java agent blurb under that doc so we can reference that other places. I removed features that weren't relevant in general.